src/data/models/Happiness.ts

```diff
@@ -8,6 +8,12 @@ export interface Happiness {
   author: User;
 }
 
+export interface NewHappiness {
+  value: number;
+  comment: string;
+  timestamp: string;
+}
```

Comment:
What is this interface used for? Add documentation. Why the name `NewHappiness`?